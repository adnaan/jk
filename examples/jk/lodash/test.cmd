npm install lodash-es@4.17.11
jk run lodash.js
