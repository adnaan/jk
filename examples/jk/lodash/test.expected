{
  "a": 1,
  "b": 2
}
[
  [
    1,
    3
  ],
  [
    2,
    4
  ]
]
